sdk/tables/Azure.Data.Tables/tests/TableServiceLiveTestsBase.cs

```diff
@@ -97,6 +82,18 @@ public async Task TablesTestSetup()
                 _ => throw new NotSupportedException("Unknown endpoint type")
             };
 
+            ConnectionString = _endpointType switch
+            {
+                TableEndpointType.Storage => $"DefaultEndpointsProtocol=https;AccountName={AccountName};AccountKey={AccountKey};EndpointSuffix={TestEnvironment.StorageEndpointSuffix};",
```

Comment:
Does ARM return a connection string for deployment? Might be a more real-world test to use the actual portal provided connection strings instead of generated ones.

Reply:
I think I'll just remove it for now.